apps/test/unit/templates/studioHomepages/SectionsTableTest.js

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import {expect} from '../../../util/configuredChai';
+import i18n from '@cdo/locale';
+import {joinedSections} from './homepagesTestData';
+import SectionsTable from '@cdo/apps/templates/studioHomepages/SectionsTable';
+
+describe('SectionsTable', () => {
+
+  it('shows column headers for students with teacher-managed accounts', () => {
+    const wrapper = shallow(
+      <SectionsTable
+        sections={joinedSections}
+        isRtl={false}
+        isTeacher={false}
+        canLeave={false}
+      />
+    );
+    expect(wrapper).to.containMatchingElement(
+      <td>
+        <div>
+          {i18n.section()}
+        </div>
+      </td>
+    );
+    expect(wrapper).to.containMatchingElement(
+      <td>
+        <div>
+          {i18n.course()}
+        </div>
+      </td>
+    );
+    expect(wrapper).to.containMatchingElement(
+      <td>
+        <div>
+          {i18n.teacher()}
+        </div>
+      </td>
+    );
+    expect(wrapper).to.containMatchingElement(
+      <td>
+        <div>
+          {i18n.sectionCode()}
+        </div>
+      </td>
+    );
+    // Students with teacher-managed accounts should not see a button to leave the section.
+    expect(wrapper.find('Button').exists()).to.be.false;
+  });
+
+  it('shows column headers for students who do not have teacher-managed accounts', () => {
+    const wrapper = shallow(
+      <SectionsTable
+        sections={joinedSections}
+        isRtl={false}
+        isTeacher={false}
+        canLeave={true}
+      />
+    );
+    expect(wrapper).to.containMatchingElement(
```

Comment:
The set of headers in this second case is the same as the first case - only the presence of the `Button` changes.  Once breaking apart into two tests, maybe the second headers test isn't needed?